<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KIT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 242, 242);
color: rgb(71, 89, 86);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>80</y>
     <width>380</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="btnFile">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>9</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;저장된 파일&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;나눔고딕&quot;;
background-color: rgb(250, 251, 250);</string>
      </property>
      <property name="text">
       <string>Search File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnYoutube">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>9</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;유튜브 링크&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;나눔고딕&quot;;
background-color: rgb(250, 251, 250);</string>
      </property>
      <property name="text">
       <string>Youtube</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>2</y>
     <width>380</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 18pt &quot;Orator Std&quot;;</string>
   </property>
   <property name="text">
    <string>Keyword In The video</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>268</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;나눔고딕&quot;;</string>
   </property>
   <property name="text">
    <string>※ 유튜브 링크인지, 저장된 파일인지 선택하세요.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lblAPIKey">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>API Key:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtAPIKey">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255,255,255);
font: 63 10pt &quot;나눔고딕&quot;;
color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="btnGetKey">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(180, 197, 193);
font: 75 11pt &quot;Arial&quot;;
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
color: rgb(51, 69, 66);</string>
        </property>
        <property name="text">
         <string>Get Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;나눔고딕&quot;;</string>
   </property>
   <property name="text">
    <string>※ 발급받은 본인의 API Key를 입력해주세요. 발급받으시려면         를 클릭하세요.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnETRI">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>278</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
color: rgb(0, 0, 255);
font: 8pt &quot;나눔고딕&quot;;
border-style:flat</string>
   </property>
   <property name="text">
    <string>여기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;&lt;주의!&gt;&gt;
 네트워크 연결이 되어있어야 합니다.
도중에 네트워크 연결이 끊길 경우 프로그램이 종료됩니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
